app/scripts/ui.js

```diff
@@ -1,3 +1,4 @@
+const Root = require('../../ui/app/pages')
```

Comment:
Why do we need to requrie `Root` here and pass it to start pop-up when the file where `startPopup` is defined was previously importing/requiring `Root`?

Reply:
We need a new entry point to mount a different `Root` for notification view.  `notification.js` requires `NotificationRoot` instead, creating a smaller bundle.